### DISCUSSION TITLE
A type annotation is necessary when exporting createSafeActionClient()

### DISCUSSION THREAD

**Discussion by zealerzone** [next-safe-action/next-safe-action#187] [General]
I using the latest version of next-safe-action: 7.1.3.

When i try to export as below, i am getting typescript error. is there any specific type that i can refer to to avoid the error

```
export const baseActionClient = createSafeActionClient()
```

**ERROR** : The inferred type of 'baseActionClient' cannot be named without a reference to '@root/node_modules/next-safe-action/dist/validation-errors-DAAgplxi.mjs'. This is likely not portable. A type annotation is necessary.ts(2742)

**Comment by TheEdoRan** [comment]
Have you already tried [this](https://next-safe-action.dev/docs/troubleshooting#typescript-error-in-monorepo)?

**Reply by zealerzone** [reply]
Thank you, i was checking all other discussion's and issues. Missed the Troubleshooting section ✌️
